src/sentry/testutils/helpers/datetime.py

```diff
@@ -1,15 +1,12 @@
 from __future__ import absolute_import
-from datetime import timedelta
+from datetime import datetime, timedelta
 
 __all__ = ["iso_format", "before_now"]
 
 
-from django.utils import timezone
-
-
 def iso_format(date):
     return date.isoformat()[:19]
 
 
 def before_now(**kwargs):
-    return timezone.now() - timedelta(**kwargs)
+    return datetime.utcnow() - timedelta(**kwargs)
```

Comment:
Is this more correct? I thought we generally used `timezone.now()` around the codebase

Reply:
For the most part we do. Where we don't is when we talk to snuba (which uses its own 'now') and in acceptance tests where timezone.now() is frozen to 2013.